core/core.js

```diff
@@ -0,0 +1,164 @@
+'use strict'; // For testing with node.js
+
+// Global scope declarations:
+var $ = Object.create(Object.prototype);
+
+(function() {
+  // Core build options:
+
+  // Overwrite existing?:
+  var force = false;
+
+  /**
+   * Add a property to a core object - but only overwrite existing
+   * properties if force is true.
+   * @param {Object} obj The object to which to add a property.
+   * @param {string} key The property key to add.
+   * @param {*} value The property key to add.
+   */
+  function make(obj, key, value) {
+    if (force || typeof obj[key] === 'undefined') {
+      obj[key] = value;
+    }
+  }
+
+  /*******************************************************************
+   * $.object
+   */
+  make($, 'object', Object.prototype);
+
+  /*******************************************************************
+   * $.physical
+   */
+  make($, 'physical', Object.create($.object, {
+    name: {value: 'Physical object prototype',
+	   writable: true, enumerable: true, configurable: true},
```

Comment:
In the case of true/true/true, why not just assign the property normally?
That said, should these be enumerable?